Jenkinsfile

```diff
@@ -0,0 +1,154 @@
+import groovy.json.JsonOutput
+
+/** Tox environment */
+def environment = 'tests'
```

Comment:
There is no tox environment named `tests`. The environments are: `py27`, `py35`, `py27-marionette`, and `py35-marionette`. For now I think it's fine to limit Jenkins to `py27` as that's what we're currently running.